package.json

```diff
@@ -49,7 +49,7 @@
         "range-parser": "^1.2.1",
         "type-is": "^1.6.18",
         "typed-emitter": "^2.1.0",
-        "uWebSockets.js": "github:uNetworking/uWebSockets.js#v20.43.0"
+        "uWebSockets.js": "github:uNetworking/uWebSockets.js#v20.44.0"
```

Comment:
it seems that the last version of uWebSockets.js is 20.63.0
Shouldn't we update to the lastest actual version?

Reply:
Yes , didn’t see that.

Reply:
Yes, sorry, my bad

Reply:
Yep, thanks guys 🙌